recipes/divemesh/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+make -j "${CPU_COUNT}"
```

Comment:
You probably don't need a patch but could use this
```suggestion
make -j "${CPU_COUNT}" CXX=$CXX LDFLAGS=$LDFLAGS CXXFLAGS=$CXXFLAGS
```

Reply:
I found errors when including `LDFLAGS` and `CXXFLAGS`, so I've kept the upstream versions of these. But I've dropped the patch in favor of `CXX=$CXX`, which is a much cleaner solution.